java/src/hu/holyoil/skeleton/testcases/SettlerMinesUraniumWithFullStorage.java

```diff
@@ -0,0 +1,78 @@
+package hu.holyoil.skeleton.testcases;
+
+import hu.holyoil.crewmate.Settler;
+import hu.holyoil.neighbour.Asteroid;
+import hu.holyoil.resource.Uranium;
+import hu.holyoil.skeleton.Logger;
+import hu.holyoil.skeleton.TestCase;
+import hu.holyoil.storage.PlayerStorage;
+
+public class SettlerMinesUraniumWithFullStorage extends TestCase {
+    private Uranium u;
+    private Uranium u0,u1,u2,u3,u4,u5,u6,u7,u8,u9;
+    private Asteroid a;
+    private Settler s;
+    private PlayerStorage ps;
```

Comment:
Only `Asteroid a` and `Settler s` need to be class fields, as they're called in `start()` method, all the others may be local variables.

Also, you can use a `List<Uranium>` for storing the Uraniums, no need to go against DRY.